.github/scripts/stats/fetch_update_gist.sh

```
@@ -0,0 +1,14 @@
+CLONES=$(curl -s -H "Authorization: token $LIGHTVM_PAT" \
+"https://api.github.com/repos/$GITHUB_REPOSITORY/traffic/clones" \
+| jq '.count // 0')
+
+TRAFFIC_DATA=$(curl -s -H "Authorization: token $LIGHTVM_PAT" \
+"https://api.github.com/repos/$GITHUB_REPOSITORY/traffic/views")
+
+TOTAL_VIEWS=$(echo $TRAFFIC_DATA | jq '.count // 0')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)